set.mm

```diff
@@ -706009,6 +706084,56 @@ not even needed (it can be any class).  (Contributed by Glauco
       ( wcel wral wf ralrimia fmptff sylib ) ADELZBCMCEFNARBCGJOBCEDFHIKPQ $.
   $}
 
+  ${
+    fvmpt2df.1 $e |- F/_ x A $.
+    fvmpt2df.2 $e |- F = ( x e. A |-> B ) $.
+    fvmpt2df.3 $e |- ( ( ph /\ x e. A ) -> B e. V ) $.
+    $( Deduction version of ~ fvmpt2 .  (Contributed by Glauco Siliprandi,
+       24-Jan-2025.) $)
+    fvmpt2df $p |- ( ( ph /\ x e. A ) -> ( F ` x ) = B ) $=
+      ( cv wcel wa cfv cmpt fveq1i wceq id fvmpt2f syl2an2 eqtrid ) ABJZCKZLUAE
+      MUABCDNZMZDUAEUCHOUBUBADFKUDDPUBQIBCDFGRST $.
+  $}
+
+  ${
+    $d A a t v w x y z $.  $d J a t w x y z $.  $d V a t w x y z $.
+    rn1st.1 $e |- F/_ x B $.
+    $( The range of a function with a first-countable domain is itself
+       first-countable.  This is a variation of ~ 1stcrestlem , with a
+       essential non-free condition replacing a disjoint variable constraint.
+       (Contributed by Glauco Siliprandi, 24-Jan-2025.) $)
+    rn1st $p |- ( B ~<_ _om -> ran ( x e. B |-> C ) ~<_ _om ) $=
+      ( com cdom wbr cmpt crn cdm ccrd wcel wfo wss con0 word cvv mpancom domtr
+      mpisyl ordom wb reldom brrelex2i elong mpbiri ondomen eqid dmmptssf ssnum
+      syl sylancl wfun funmpt funforn mpbi fodomnum ctex ssdomg syl2anc ) BEFGZ
+      ABCHZIZVBJZFGZVDEFGZVCEFGVAVDKJZLZVDVCVBMZVEVABVGLZVDBNZVHEOLZVAVJVAVLEPZ
+      UAVAEQLVLVMUBBEFUCUDEQUEUKUFEBUGRABCVBDVBUHUIZBVDUJULVBUMVIABCUNVBUOUPVDV
+      CVBUQTVDBFGZVAVFVABQLVKVOBURVNVDBQUSTVDBESRVCVDESUT $.
+  $}
+
+  ${
+    rnmptssff.1 $e |- F/_ x A $.
+    rnmptssff.2 $e |- F/_ x C $.
+    rnmptssff.3 $e |- F = ( x e. A |-> B ) $.
+    $( The range of an operation given by the maps-to notation as a subset.
```

Comment:
"The range of an operation ..." -> "The range of a function ..." (also in the following comments)

Reply:
[Commit e51bcfc](https://github.com/metamath/set.mm/pull/4606/commits/e51bcfc538c744a907eed6fe63ae3afff90f50f9) should have fixed this